firebase-fetcher/build.gradle

```diff
@@ -17,10 +18,17 @@ android {
     }
 }
 
+repositories {
+    mavenCentral()
+}
+
 dependencies {
-    testImplementation 'junit:junit:4.12'
-    implementation 'com.google.firebase:firebase-config:19.0.3'
+    testImplementation 'junit:junit:4.13'
+    implementation 'com.google.firebase:firebase-config:19.1.4'
     implementation project(':status-resolver')
+    implementation "androidx.core:core-ktx:+"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'org.jetbrains.kotlinx:kotlinx-coroutines-play-services:1.3.3'
```

Comment:
there is `1.3.7` version of coroutines.

Reply:
1625e84cb028ce7b65cc6bd34201660fe27780f8